### ISSUE TITLE
[Popover] Add animation

### ISSUE THREAD

**Issue by pawel-twardziak** [radix-ng/primitives#196]
### Overview

showing, hidding, repositioning should be animated

### Examples in other libraries

Radix UI popover is being animated https://www.radix-ui.com/primitives/docs/components/popover

### Who does this impact? Who is this for?

Users - poorer UX

**Comment by pawel-twardziak** [comment]
@pimenovoleg for your consideration

**Comment by pimenovoleg** [comment]
Yes, it's a great feature, we've adapted it for the accordion and some other elements. 

It's important to try to maintain the ability to control animation through CSS variables, as it provides flexibility in integration with Tailwind.

**Comment by pawel-twardziak** [comment]
I see it was animated after I initially copied all the stuff from `tooltip` but during the refactors I've removed it.
I will bring back the animations.
